test/avocado/selenium-login.py

```diff
@@ -134,23 +174,24 @@ def login(self, tmpuser=user, tmppasswd=passwd):
         elem = self.wait_id('login-password-input')
         elem.clear()
         elem.send_keys(tmppasswd)
-        self.wait_id("login-button").click()
+        self.click(self.wait_id("login-button", cond=clickable))
         return elem
```

Comment:
Do we need to return `elem`? It is assigned in some calls, but never used as far as I can tell.